rotkehlchen/tests/unit/decoders/test_makerdao_sai.py

```diff
@@ -0,0 +1,757 @@
+import pytest
+
+from rotkehlchen.accounting.structures.balance import Balance
+from rotkehlchen.accounting.structures.base import HistoryBaseEntry
+from rotkehlchen.accounting.structures.types import HistoryEventSubType, HistoryEventType
+from rotkehlchen.assets.asset import Asset
+from rotkehlchen.chain.ethereum.modules.makerdao.sai.constants import CPT_SAI
+from rotkehlchen.chain.evm.decoding.constants import CPT_GAS
+from rotkehlchen.chain.evm.structures import EvmTxReceipt, EvmTxReceiptLog
+from rotkehlchen.chain.evm.types import string_to_evm_address
+from rotkehlchen.constants.assets import A_ETH, A_SAI, A_WETH
+from rotkehlchen.db.evmtx import DBEvmTx
+from rotkehlchen.fval import FVal
+from rotkehlchen.tests.utils.ethereum import get_decoded_events_of_transaction
+from rotkehlchen.types import ChainID, EvmTransaction, Location, deserialize_evm_tx_hash
+from rotkehlchen.utils.hexbytes import hexstring_to_bytes
+
+A_PETH = Asset('eip155:1/erc20:0xf53AD2c6851052A81B42133467480961B2321C09')
+
+
+@pytest.mark.parametrize('ethereum_accounts', [['0x01349510117dC9081937794939552463F5616dfb']])  # noqa: E501
+def test_makerdao_sai_new_cdp(ethereum_transaction_decoder):
```

Comment:
I need you to change all tests in this file to have the events hardcoded. I don't want any more network calls inside the tests.